litmus-portal/frontend/cypress/component/LinearProgressBar.spec.tsx

```diff
@@ -4,20 +4,36 @@ import { mount } from 'cypress-react-unit-test';
 import LinearProgressBar from '../../src/components/ProgressBar/LinearProgressBar';
 
 describe('Linear Progressbar Testing', () => {
-  it('Progressbar color is Red for value 3', () => {
+  it('Progressbar stroke for value 3', () => {
     mount(<LinearProgressBar value={2} />);
```

Comment:
In it(), you have given that you are testing for value 3, but during mounting the component,  the value given is 2?

Reply:
Thanks for the catch @Jonsy13 . I'll make the changes!